include/decimalfield.h

```diff
@@ -149,7 +149,7 @@ class DecimalField : public Field
      */
     virtual operator float() const override
     {
-        return static_cast<float>(_number / pow(10, _places));
+        return static_cast<float>(_number / pow(10., _places));
```

Comment:
I think it's better to avoid casting vetween float and double and just use float literal like `10.0f` or directly select specialized version of pow like `pow<float>()` here